components/style/properties/longhand/pointing.mako.rs

```diff
@@ -6,142 +6,14 @@
-    }
-
-    #[cfg(not(feature = "gecko"))]
-    #[inline]
-    pub fn get_initial_value() -> computed_value::T {
-        computed_value::Keyword::Auto
-    }
-
-    #[cfg(feature = "gecko")]
-    #[inline]
-    pub fn get_initial_value() -> computed_value::T {
-        computed_value::T {
-            images: vec![],
-            keyword: computed_value::Keyword::Auto
-        }
-    }
-
-    impl Parse for computed_value::Keyword {
-        fn parse<'i, 't>(_context: &ParserContext, input: &mut Parser<'i, 't>)
-                         -> Result<computed_value::Keyword, ParseError<'i>> {
-            #[allow(unused_imports)] use std::ascii::AsciiExt;
-            use style_traits::cursor::Cursor;
-            let location = input.current_source_location();
-            let ident = input.expect_ident()?;
-            if ident.eq_ignore_ascii_case("auto") {
-                Ok(computed_value::Keyword::Auto)
-            } else {
-                Cursor::from_css_keyword(&ident)
-                    .map(computed_value::Keyword::Cursor)
-                    .map_err(|()| location.new_custom_error(SelectorParseErrorKind::UnexpectedIdent(ident.clone())))
-            }
-        }
-    }
-
-    #[cfg(feature = "gecko")]
-    fn parse_image<'i, 't>(context: &ParserContext, input: &mut Parser<'i, 't>)
-                           -> Result<computed_value::Image, ParseError<'i>> {
-        Ok(computed_value::Image {
-            url: SpecifiedUrl::parse(context, input)?,
-            hotspot: match input.try(|input| input.expect_number()) {
-                Ok(number) => Some((number, input.expect_number()?)),
-                Err(_) => None,
-            },
-        })
-    }
-
-    #[cfg(not(feature = "gecko"))]
-    pub fn parse<'i, 't>(context: &ParserContext, input: &mut Parser<'i, 't>)
-                         -> Result<SpecifiedValue, ParseError<'i>> {
-        computed_value::Keyword::parse(context, input)
-    }
-
-    /// cursor: [<url> [<number> <number>]?]# [auto | default | ...]
-    #[cfg(feature = "gecko")]
-    pub fn parse<'i, 't>(context: &ParserContext, input: &mut Parser<'i, 't>)
-                         -> Result<SpecifiedValue, ParseError<'i>> {
-        let mut images = vec![];
-        loop {
-            match input.try(|input| parse_image(context, input)) {
-                Ok(mut image) => {
-                    image.url.build_image_value();
-                    images.push(image)
-                }
-                Err(_) => break,
-            }
-            input.expect_comma()?;
-        }
-
-        Ok(computed_value::T {
-            images: images,
-            keyword: computed_value::Keyword::parse(context, input)?,
-        })
-    }
-</%helpers:longhand>
+${helpers.predefined_type("cursor",
+                          "Cursor",
+                          "computed::Cursor::auto()",
+                          initial_specified_value="specified::Cursor::auto()",
+                          products="gecko",
```

Comment:
You need to remove this `products="gecko"`. With it you just removed the property from servo! :)